nengo_viz/server.py

```diff
@@ -11,24 +11,38 @@
     from urllib.parse import unquote
 
 import nengo_viz.swi as swi
+import nengo_viz
 
 
 class Server(swi.SimpleWebInterface):
     """Web server interface to nengo_viz"""
 
     def swi_browse(self, dir):
```

Comment:
This isn't part of this PR, but we really shouldn't be using the variable `dir` since it shadows the builtin `dir`.